### ISSUE TITLE
On Refresh Cache, Bitbucket tab dissappears

### ISSUE THREAD

**Issue by abaumer** [afragen/github-updater#715]
**Issue**
Github updater looses bitbucket tab after clicking Refresh Cache, then doesn't look for theme or plugin updates. Sad.

**Steps to reproduce**
After installing a theme or plugin with Bitbucket Repo URI from WP plugin/theme install OR from GHU install tabs, the plugin/theme installs fine. Then from GHU settings page the Bitbucket tab is visible and appears to have correct information. Then clicking Refresh Cache, the BB tab disappears and no themes or plugins seem to check for updates. Manually reducing the version on the theme/plugin to force GHU to show updates, then from GHU settings clicking Refresh Cache produces no updates. 

It's as if something is hanging the first time Refresh Cache is clicked, then preventing it from completing and preventing future updates.

**Setup**
Multisite Running on Flywheel

**Plugins** 
GHU only

**Themes**
Twenty Seventeen only

**Other Notes**
Also get these notices when Viewing Details on the Plugin in question:
Notice: Undefined property: stdClass::$dot_org in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 351 
Notice: Undefined property: stdClass::$donate_link in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 360 
Notice: Undefined property: stdClass::$remote_version in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 361 
Notice: Undefined property: stdClass::$requires in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 363 
Notice: Undefined property: stdClass::$requires_php in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 364 
Notice: Undefined property: stdClass::$tested in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 365 
Notice: Undefined property: stdClass::$downloaded in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 366 
Notice: Undefined property: stdClass::$last_updated in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 367 
Notice: Undefined property: stdClass::$download_link in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 368 
Notice: Undefined property: stdClass::$contributors in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Plugin.php on line 371

Do also have these Sunrise related Warnings. And Flywheel team is looking into it.:
Warning: include_once(/www/wp-content/sunrise.php): failed to open stream: No such file or directory in /www/wp-includes/ms-settings.php on line 47 
Warning: include_once(): Failed opening '/www/wp-content/sunrise.php' for inclusion (include_path='.:/usr/share/php:/www/wp-content/pear') in /www/wp-includes/ms-settings.php on line 47 

And have these PHP notice/warnings:
[13-Sep-2018 18:26:41 UTC] PHP Notice:  Undefined index: github_updater in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Settings.php on line 610
[13-Sep-2018 18:26:41 UTC] PHP Warning:  array_merge(): Argument #2 is not an array in /www/wp-content/plugins/github-updater/src/GitHub_Updater/Settings.php on line 610

**Things I've tried**
- installing themes/plugins from Wordpress admin with BB Repo URI
- installing themes/plugins from GHU install tabs
- installing only the theme
- installing only the plugin
- using BB username and email
- deactivating all other plugins and themes
- refreshing the browser window many times after theme/plugin install on GHU settings, but BB tab stil shows until you click Refresh Cache
- Looking through PHP logs for other 
- Attempting this many times over.
- Confirming and Reconfirming the BB Repo URI for theme and plugin (Lots of times)

**Known differences from all other sites using GHU**
Multisite
Has Sunrise stuff installed

Any hot tips are appreciated.

**Comment by afragen** [comment]
The tab disappears until WP-Cron fires off a cron.php request. This is temporary and reloading the page or looking at other pages should cause this to fire. 

https://github.com/afragen/github-updater/wiki/Background-Processing

**Comment by afragen** [comment]
The other undefined property warnings are because you clicked on the View Details of a plugin that has had its info removed from the Refresh Cache but hasn’t yet been updated from wp-cron.

**Comment by abaumer** [comment]
Thanks for the follow up and notes @afragen. I did look into both items, gave it time, and still got nothing, no tabs returned and no theme nor plugin updates showed, even after days. 

I updated to 8.3.1 and the issue seems to be resolved! 
The Cache refreshes quickly and the tabs don't disappear. The theme shows updates quickly and correctly updates. Whatever the boostrap file update did, it helped. 

Thanks!
